enhancements/console/dynamic-plugins.md

```diff
@@ -272,6 +272,35 @@ available and the user must refresh their browser to see the changes. (We won't
 refresh the page automatically to avoid possibly losing data if the user is
 entering something into a form or the YAML editor.)
 
+### Localization
+
+Console's frontend will use `/locales/resource.json?lng={{lng}}&ns={{ns}}`
+as i18next loadPath.
+
+#### Dynamic Plugins
+
+Dynamic plugins will need to contain their own localization files and serve them
+per requested. All dynamic plugins must use a single localization namespace
+per language, called `default.json`.
```

Comment:
I think we'd want the namespace to match whatever the code is using (so if it's `plugin:dev-console` that filename should match). We might be careful with special characters like `:`... Maybe something like `plugin__` is a better prefix if we want to avoid conflicts?

Reply:
Right, but I think using a different namespace in the code compare to the filename is confusing and not how i18next typically works. Simplest would be to make them the same IMO.

Reply:
I'm indifferent to use default or the plugin name itself.

As for the `:`, what's wrong with it as a separator? Certainly read better imo.

Reply:
> I'm indifferent to use default or the plugin name itself.

+1

> As for the :, what's wrong with it as a separator? Certainly read better imo.

yeah I think its more readable as well. not sure what is the issue

Reply:
It's the same reason I wouldn't recommend putting a space in the filename if it can be easily avoided. It needs to be escaped in URLs, and we need to be sure i18next and any dev tools we use handle special characters correctly. Have we tested that i18next-http-backend properly URL encodes `:` in the `loadPath`? Using underscores instead sidesteps the problem entirely.

I actually find `__` more readable as a separator because it gives more space between the prefix and actual name. Maybe I'm just used to BEM classnames at this point.